x-pack/plugins/lens/public/datatable_visualization/components/dimension_editor.tsx

```diff
@@ -175,7 +202,61 @@ export function TableDimensionEditor(
           />
         </EuiFormRow>
       )}
-      {isNumericField && (
+      {isNumeric && (
+        <>
+          <EuiFormRow
+            fullWidth
+            label={i18n.translate('xpack.lens.table.summaryRow.label', {
+              defaultMessage: 'Summary Row',
+            })}
+            display="columnCompressed"
+          >
+            <EuiComboBox
+              fullWidth
+              compressed
+              isClearable={false}
+              data-test-subj="lnsDatatable_summaryrow_function"
+              placeholder={i18n.translate('xpack.lens.indexPattern.fieldPlaceholder', {
+                defaultMessage: 'Field',
+              })}
+              options={getSummaryRowOptions()}
+              selectedOptions={[
+                {
+                  label: fallbackSummaryLabel,
```

Comment:
This logic is picking up the user supplied summary label as the selected summary function:
<img width="441" alt="Screenshot 2021-06-08 at 14 17 54" src="https://user-images.githubusercontent.com/1508364/121183763-b3936c00-c864-11eb-9ff3-2c757fc29796.png">


Especially confusing for no label:
<img width="446" alt="Screenshot 2021-06-08 at 14 17 29" src="https://user-images.githubusercontent.com/1508364/121183801-bd1cd400-c864-11eb-8b66-e4352206da4b.png">